Automattic/jetpack
# VideoPress: Add option to load admin UI

#### Changes proposed in this Pull Request:

* Adds a way to choose whether we want to initialize the VideoPress admin UI or not, and initialize it from the package

#### Other information:


#### Jetpack product discussion


NA

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Activate Jetpack and the VideoPress module
* Make sure you don't see the Jetpack > VideoPress menu in the admin menu
* Activate the VideoPress plugin
* Make sure you see the menu item now
* Deactivate Jetpack and confirm you still see the menu item



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202813985335436
